Sources/MapboxMaps/Foundation/Extensions/Core/ResourceOptions.swift

```diff
@@ -22,42 +31,36 @@ public extension ResourceOptions {
                   cacheSize: NSNumber(value: cacheSize))
     }
 
-    var cacheSize: UInt64? {
+    public var cacheSize: UInt64? {
```

Comment:
Why was this made public?

Reply:
previously, the entire extension was public, I moved the keyword down as was the pattern